### PR TITLE
fix: upload app fixed after api changes

### PR DESCRIPTION
**Pull request by edge33**
### Description

- Done adapting upload app to new sdk

## Type

- [ ] **Dependency upgrade**
- [x] **Bug fix**
- [x] **New feature**
- [x] **Dev change**
- [ ] **Additional tests**
- [ ] **Documentation**
- [ ] **Other**

### Checklist:

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] This change might impact the developer experience of others and should be communicated

### Screenshots:

N/A

### CODE REVIEW

**Review by rgrnearform — APPROVED** [review]

File: apps/upload/app/utils/logto-config.ts [hunk 1]
```diff
@@ -9,29 +9,35 @@ export const uploadApiResource = process.env.UPLOAD_BACKEND_URL?.endsWith("/")
...
-};
+const uploadEntryPoint = (
+  process.env.NEXT_PUBLIC_UPLOAD_SERVICE_ENTRY_POINT?.endsWith("/")
+    ? process.env.NEXT_PUBLIC_UPLOAD_SERVICE_ENTRY_POINT.substring(
+        0,
+        process.env.NEXT_PUBLIC_UPLOAD_SERVICE_ENTRY_POINT.length - 1,
+      )
+    : process.env.NEXT_PUBLIC_UPLOAD_SERVICE_ENTRY_POINT
+) as string;
+const uploadEntryPointSlash = `${uploadEntryPoint}/` as string;
```
Comment (matteolc) [inline apps/upload/app/utils/logto-config.ts:24]: You might make life easier by using built in `node:url` methods?

```
const url = require('node:url')
const myURL = new URL('https://a.com/');
myURL.origin = 'https://a.com'
```

File: apps/upload/app/utils/building-blocks-client.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  default as getBuildingBlockSDK,
+} from "@ogcio/building-blocks-sdk";
+import { headers } from "next/headers";
+
+let buildingBlockSdk: BuildingBlocksSDK | undefined = undefined;
+
+export const getSdks = () => {
+  // if (buildingBlockSdk) {
+  //   return buildingBlockSdk;
+  // }
```
Comment (matteolc) [inline apps/upload/app/utils/building-blocks-client.ts:12]: Probably want to remove this?

**Review by matteolc — APPROVED** [review]
LGTM just some minor comments

**Comment by SamSalvatico** [comment]
> LGTM just some minor comments - feel free to resolve the convos if they are blocking!

Thank you @matteolc !
I've updated the code given that @edge33 is AFK, could you check it again?

**Review by matteolc — APPROVED** [review]
LGTM

**Review by matteolc — APPROVED** [review]

**Review by matteolc — DISMISSED** [review]

**Pull request merged**